### PR TITLE
Update package.json to support react 17

### PR DESCRIPTION
**Pull request by phuhl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,8 +72,8 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "react": "~16",
-    "react-dom": "~16",
+    "react": "~17",
```
Comment (JonatanSalas) [inline package.json:75]: We should do something like this: `"react": "^16.8.0 || ^17.0.0"`, the way you're doing affects users running React 16, same with react-dom `"react-dom": "^16.8.0 || ^17.0.0"`

**Review by JonatanSalas — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (phuhl) [reply package.json:75]:
```suggestion
    "react": "~17 || ^16.8.0",
```

File: package.json [hunk 2]
```diff
@@ -72,8 +72,8 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "react": "~16",
-    "react-dom": "~16",
+    "react": "~17 || ^16.8.0",
+    "react-dom": "~17",
```
Comment (phuhl) [inline package.json:76]:
```suggestion
    "react-dom":  "~17 || ^16.8.0",
```

**Comment by JonatanSalas** [comment]
Hi, I've moved the code from master to a v3 branch because we're going to merge the v4. I've updated the code and released the v3.1.0, you can install it by doing `npm i @blackbox-vision/react-qr-reader@3.1.0`

**Pull request closed without merging**